src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -28,7 +31,7 @@
     public void constructor() {
         assertEquals(new UserPrefs(), modelManager.getUserPrefs());
         assertEquals(new GuiSettings(), modelManager.getGuiSettings());
-        assertEquals(new AddressBook<Patient>(), new AddressBook<>(modelManager.getPatientRecords()));
+        assertEquals(new AddressBook<Patient>(), new AddressBook<>(modelManager.getDoctorRecords()));
```

Comment:
should not be a remove and an add, and line 34 looks like it should fail: comparing `AddressBook<Patient>` with `AddressBook<Doctor>`

Reply:
@Jacob-Pang any ideas on why this isn't failing tests?

Reply:
It is because the initialised address book is empty here.
Should have one for patients and doctors sorry

Reply:
shouldn't the typing of addressbook fail the equals?